CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.1.0
+
+2018-05-15
```

Comment:
Please don't write changelogs for releases in the past ;)

Just leave that kind of stuff to me, it can be confusing for people to see releases in the changelog that don't actually exist.